### DISCUSSION TITLE
2022-09-22

### DISCUSSION THREAD

**Discussion by Woon-2** [MyeongWoonJang/cpp-study#11] [Show and tell]
## 어제 한 일
어제는
[Github Desktop Guide](https://github.com/Woon-2/cpp-study/blob/main/readme/github_desktop_guide.md)의
- `Fetch`
- `Pull`
- `Commit`
- `Push`
- `Amend Commit`
- `Revert Commit`

항목을 작성했습니다.

## 오늘 할 일
오늘은 같은 게시글을 이어 작성할 것입니다.
- `Create Branch` 항목
- `Checkout Branch` 항목

**Comment by Lcat12345** [comment]
수고하셨습니다!
